### ISSUE TITLE
Feature request: Nextcloud News API support

### ISSUE THREAD

**Issue by aadilayub** [Ashinch/ReadYou#8]
It'd be great to be able to use this app in sync with the Nextcloud News RSS reader!

**Comment by Ashinch** [comment]
The basic functionality is currently being refined, and the higher priority FreshRSS API will be supported later, with support for the other APIs coming after that.

**Comment by daffydock** [comment]
That sounds fair. 
For what is worth, when resources are available, I would second the opportunity to have the app support the Nextcloud RSS API. 
Thank you.

**Comment by 0-st** [comment]
Would it also be possible to support Miniflux?

**Comment by sheeepdev** [comment]
> Would it also be possible to support Miniflux?

I agree, Miniflux should be supported because it is also highly used in the self-hosted open-source RSS reader space

**Comment by yavko** [comment]
This would be amazing, since regular news app isn't md3

**Comment by gregorni** [comment]
Can we already put Nextcloud in the list of planned integrations in the Readme, just so that it's there?

**Comment by aadilayub** [comment]
> Can we already put Nextcloud in the list of planned integrations in the Readme, just so that it's there?

that should be up to the maintainer imo. they don't have to work on this

**Comment by Saijin-Naib** [comment]
Just adding a plus one as I'm setting up my own NextCloud, and I'm quite fond of Read You, so I'd love to see sync across devices using my NextCloud.

**Comment by dontobi** [comment]
I would also appreciate the support of Nextcloud. I like ReadYou very much, but I need multi-device support (via sync via Nextcloud).

**Comment by Eschguy** [comment]
Any status updates on the Nextcloud support?
